README.md

```diff
@@ -265,10 +263,6 @@ hvd.init()
 # Verify that MPI multi-threading is supported.
 assert hvd.mpi_threads_supported()
 
-# Make sure MPI is not re-initialized.
-import mpi4py.rc
-mpi4py.rc.initialize = False
-
 from mpi4py import MPI
```

Comment:
I think this will actually fail because `hvd.init()` will initialize MPI. We should bring MPI import before `hvd.init()`

Reply:
I have tried it, it doesn't fail because mpi4py tests if MPI has already been initialized.